packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -170,22 +172,34 @@ export const HelpCenterContactPage: FC< HelpCenterContactPageProps > = ( {
 	};
 
 	const renderChatOption = () => {
+		const productSlug = ( currentSite as HelpCenterSite )?.plan?.product_slug;
+		const plan = getPlan( productSlug );
+		const productId = plan?.getProductId();
+
+		const handleOnClick = () => {
+			contactOptionsEventMap.chat();
+
+			recordTracksEvent( 'calypso_help_live_chat_begin', {
+				site_plan_product_id: productId,
+				is_automated_transfer: currentSite?.is_wpcom_atomic,
+				force_site_id: true,
+				location: 'help-center',
+				section: sectionName,
+			} );
+
+			openChatWidget( {
+				aiChatId: wapuuChatId,
+				message: `Support request started with <strong>Wapuu</strong><br>Site: ${ currentSite?.URL }<br />Wapuu Chat: <a href="https://mc.a8c.com/odie/odie-chat.php?chat_id=${ wapuuChatId }">${ wapuuChatId }</a>`,
```

Comment:
We should encode all of these using `encodeURIComponent`

Reply:
Yes but that doesn’t necessarily imply trust. Someone may be able to create a malicious `wapuChatId` or a `currentSite.URL`.

Reply:
Cool better safe than sorry 👍